app/components/emoji/emoji.js

```diff
@@ -49,8 +50,13 @@ export default class Emoji extends React.PureComponent {
             return <Text style={textStyle}>{literal}</Text>;
         }
 
+        let ImageComponent = WebImage;
+        if (Platform.OS === 'android') {
+            ImageComponent = Image;
```

Comment:
I think it would be. The reason why I didn't make those changes in this PR is because we use a lot of the lifecycle loading events for the attachment and image previews so I need to look at the source code for that lib a little more to see how it handles those. Will definitely make a difference I think.